src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
```

Comment:
It is better use next formatting
```suggestion
  const men = century
    ? people.filter((p) => {
        return p.sex === 'm' && Math.ceil(p.died / 100) === century;
      }) 
    : people.filter((p) => {
        return p.sex === 'm';
      });
```